std/utf.d

```diff
@@ -3829,14 +3824,12 @@ template byUTF(C) if (isSomeChar!C)
                         {
                             static if (is(RC == dchar))
                             {
-                                fill = cast(ushort) encode!(Yes.useReplacementDchar)(buf, c);
                                 r.popFront;
+                                dchar dc = c;
                             }
                             else
-                            {
```

Comment:
style nitpick: If one part of the static if uses braces, all of them should use it and vice versa.